src/main/java/com/springboot/soapclient/Application.java

```diff
@@ -0,0 +1,21 @@
+package com.springboot.soapclient;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+import movies.wsdl.GetMovieByIdResponse;
+
+@SpringBootApplication
+public class Application {
```

Comment:
for simple run and check what's going on Apps you could use the command line runner. see https://www.baeldung.com/spring-boot-console-app